frontend/src/components/common/DownloadMenu.tsx

```diff
@@ -0,0 +1,54 @@
+import { SaveAlt as SaveAltIcon } from "@mui/icons-material";
+import { IconButton, Menu, MenuItem, Paper } from "@mui/material";
+import { MouseEvent, useCallback, useState } from "react";
+import { useFileExport } from "../../hooks/useFileExport";
+
+const DownloadMenu = () => {
+	const [anchorEl, setAnchorEl] = useState<HTMLElement | null>(null);
+
+	const handleClick = (event: MouseEvent<HTMLButtonElement>) => {
+		setAnchorEl(event.currentTarget);
+	};
+
+	const handleClose = () => {
+		setAnchorEl(null);
+	};
+
+	const { exportToPDF, exportToTXT, exportToDOCX } = useFileExport();
+
+	const handleExportToPDF = useCallback(() => {
+		exportToPDF();
+		handleClose();
```

Comment:
How about introducing a wrapper function? It would be good to handle the common logic in the wrapper function for the `handleExportToXXX` function.

```
// Wrapper Function
const handleExport = (onExport) => {
    onExport();
    handleClose();
}

const handleExportToPDF = () => {
    handleExport(exportToPDF)
}
```